### PR TITLE
Add timeouts for rest client

### PR DESCRIPTION
**Pull request by bbejeck**

### CODE REVIEW

**Comment by bbejeck** [comment]
ping @mjsax @guozhangwang @dguy 

Rest client timed out so setting a default timeout for all requests.
`javax.ws.rs.ServerErrorException: HTTP 504 Gateway Timeout
	at io.confluent.examples.streams.microservices.EndToEndTest.shouldCreateNewOrderAndGetBackValidatedOrder(EndToEndTest.java:58)`

File: src/test/java/io/confluent/examples/streams/microservices/EndToEndTest.java [hunk 1]
```diff
@@ -43,9 +45,9 @@
   @Test
   public void shouldCreateNewOrderAndGetBackValidatedOrder() throws Exception {
     OrderBean inputOrder = new OrderBean(id(1L), 2L, OrderState.CREATED, Product.JUMPERS, 1, 1d);
-    Client client = ClientBuilder.newClient();
+      final Client client = getClient();
```
Comment (mjsax) [inline src/test/java/io/confluent/examples/streams/microservices/EndToEndTest.java]: nit: fit indention

File: src/test/java/io/confluent/examples/streams/microservices/EndToEndTest.java [hunk 2]
```diff
@@ -43,9 +45,9 @@
...
 
-    //Add inventory required by the inventory service with enough items in stock to pass validation
+      //Add inventory required by the inventory service with enough items in stock to pass validation
```
Comment (mjsax) [inline src/test/java/io/confluent/examples/streams/microservices/EndToEndTest.java]: nit: fit indention

File: src/test/java/io/confluent/examples/streams/microservices/EndToEndTest.java [hunk 3]
```diff
@@ -61,11 +63,12 @@ public void shouldCreateNewOrderAndGetBackValidatedOrder() throws Exception {
     assertThat(returnedBean.getState()).isEqualTo(OrderState.VALIDATED);
   }
 
-  @Test
+
+    @Test
   public void shouldProcessManyValidOrdersEndToEnd() throws Exception {
-    Client client = ClientBuilder.newClient();
+        final Client client = getClient();
```
Comment (mjsax) [inline src/test/java/io/confluent/examples/streams/microservices/EndToEndTest.java]: nit: fit indention

This is all over the place -- guess you nee to change you IDE setting -- note, examples and AK uses different code style

**Comment by benstopford** [comment]
LGTM, although I think there is still a problem with the connection not being closed. I'll raise a separate PR for that.

File: src/test/java/io/confluent/examples/streams/microservices/EndToEndTest.java (hunk 3 above)
Comment (benstopford) [reply src/test/java/io/confluent/examples/streams/microservices/EndToEndTest.java]: i'll pick up fixing this in a follow up pr as I have the IDE settings configured.

**Comment by benstopford** [comment]
I've created a follow up PR (https://github.com/confluentinc/kafka-streams-examples/pull/45) 
@bbejeck remind me - is there a process for merging these changes forward up to 4.0.0 or should i just do it manually?

**Comment by bbejeck** [comment]
@benstopford 
`pint merge kafka-streams-examples --from-branch 3.3.0-post`

**Pull request merged**